pkg/cvo/metrics.go

```diff
@@ -133,7 +133,12 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 	// output cluster operator version and condition info
 	operators, _ := m.optr.clusterOperatorLister.List(labels.Everything())
 	for _, op := range operators {
-		g := m.clusterOperatorUp.WithLabelValues(op.Namespace, op.Name, op.Status.Version)
+		var firstVersion string
```

Comment:
are we assuming the operator version is going to be the first?

Reply:
No, I just didn't want to remove the metric value.  We're going to need to define it, I'll add a todo